apps/institution-service/src/models/aggregator.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  Association,
+  CreationOptional,
+  DataTypes,
+  HasManyGetAssociationsMixin,
+  InferAttributes,
+  InferCreationAttributes,
+  Model,
+  NonAttribute,
+} from "sequelize";
+import sequelize from "../database";
+import { AggregatorIntegration } from "./aggregatorIntegration";
+import { Institution } from "./institution";
+
+export class Aggregator extends Model<
+  InferAttributes<Aggregator, { omit: "aggregatorIntegrations" }>,
+  InferCreationAttributes<Aggregator, { omit: "aggregatorIntegrations" }>
+> {
+  declare id?: number;
+  declare name: string;
```

Comment:
Isn't this required to be unique in the widget? It seems like this should be the primary key